# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1295731121 18000
#      Sat Jan 22 16:18:41 2011 -0500
# Branch release-3-4-x
# Node ID 2f81402cb6cce80ccce1a056dcce0e8d7d2e5c5b
# Parent  62da62aca1890c5cebecbc657a216b3be5495405
version is now 3.3.90+

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-22  John W. Eaton  <jwe@octave.org>
+
+	* configure.ac (AC_INIT): Version is now 3.3.90+.
+
 2011-01-22  John W. Eaton  <jwe@octave.org>
 
 	* configure.ac (AC_INIT): Version is now 3.3.90.
 	(OCTAVE_API_VERSION_NUMBER): Now 44.
 	(OCTAVE_RELEASE_DATE): Now 2011-01-22.
 	(OCTAVE_COPYRIGHT): Update year to 2011.
 
 2010-01-20  Rik  <octave@nomad.inbox5.com>
diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -22,17 +22,17 @@ dnl Copyright (C) 1993-2011 John W. Eato
 
 ### Preserve CFLAGS and CXXFLAGS from the environment before doing
 ### anything else because we don't know which macros might call
 ### AC_PROG_CC or AC_PROG_CXX.
 
 EXTERN_CFLAGS="$CFLAGS"
 EXTERN_CXXFLAGS="$CXXFLAGS"
 
-AC_INIT([GNU Octave], [3.3.90], [bug@octave.org], [octave])
+AC_INIT([GNU Octave], [3.3.90+], [bug@octave.org], [octave])
 
 dnl PACKAGE_VERSION is set by the AC_INIT VERSION arg
 OCTAVE_VERSION="$PACKAGE_VERSION"
 OCTAVE_API_VERSION_NUMBER="44"
 OCTAVE_API_VERSION="api-v$OCTAVE_API_VERSION_NUMBER+"
 OCTAVE_RELEASE_DATE="2011-01-22"
 OCTAVE_COPYRIGHT="Copyright (C) 2011 John W. Eaton and others."
 AC_SUBST(OCTAVE_VERSION)
